browser/index.js

```diff
@@ -839,7 +842,12 @@ module.exports = module.exports = {
                         categories={categoriesOverall ? categoriesOverall : []}
                         onApply={_self.onApplyLayersAndChemical}
                         onClose={onCloseHandler}
-                    /></Provider>, document.getElementById(introlModalPlaceholderId));
+                    /></Provider>, document.getElementById(introlModalPlaceholderId)); */
+                ReactDOM.render(<ThemeProvider>
+                    <Dialogue titleText='Welcome to Calypso'
```

Comment:
Use the i18n language translation from Vidi/Calypso (there are some, somewhere :-))